lib/board.ex

```diff
@@ -12,4 +12,53 @@ defmodule Board do
   def place_a_symbol(board, player_move, symbol) do
     Map.replace(board, String.to_integer(player_move), symbol)
   end
+
+  @spec has_player_won(map()) :: boolean()
+  def has_player_won(board) do
+    Enum.any?([get_all_rows_status(board), get_all_columns_status(board), get_both_diagonals_status(board)])
+  end
+
+  @spec get_board_dimensions(map()) :: integer()
+  def get_board_dimensions(board) do
+    trunc(:math.sqrt(Enum.count(board)))
+  end
```

Comment:
Can you add unit tests for this? If you replace the implementation here with a hard-coded value of `3` do any of your tests fail?

Reply:
Absolutely! I was doing this right after I pushed the last commit because I remembered us talking about this but completely forgot for the push 😂 I tried replacing line 23 with the number 3 and it doesn't seem like any tests fail! 🥳

Reply:
I think we might be lacking a little test coverage if a hard-coded `3` doesn't fail any tests 🙂